ui/pages/confirmations/components/fee-details-component/fee-details-component.js

```diff
@@ -168,18 +173,15 @@ export default function FeeDetailsComponent({
                   {t('layer1Fees')}
                 </Text>
               }
-              detailText={
-                useCurrencyRateCheck && renderTotalDetailText(layer1GasFee)
```

Comment:
`useCurrencyRateCheck` is already checked in `getShouldShowFiat` selector hence am removing this.